### DISCUSSION TITLE
Commit author is always Administrator when using git outside WebUI

### DISCUSSION THREAD

**Discussion by mathieufrh** [requarks/wiki#3312] [Error / Bug Report]
**Describe the bug**
When using git to commit and push, WikiJS shows "Administrator" as commit author.

**To Reproduce**
1. Configure WikiJS with git in bi-directional mode
2. Clone the repository
3. Edit something, commit and push
4. WikiJS will pull the repo but shows your commit as being authored by "Administrator"

**Expected behavior**
WikiJS use the author from the commit on the repository.

**Host Info (please complete the following information):**
 - OS: Debian 10.5
 - Wiki.js version: 2.5.170
 - Database engine: postgres 11+200+deb10u3

**Comment by Chew** [comment]
I'm getting this as well. Would be nice to attribute changes to people even just based on the Git name
